entity_rsa_checker/main.gs

```diff
@@ -1,108 +1,309 @@
-/*
-    Name:        WeDiscover - RSA Checker, Google Ads Script
```

Comment:
Is this doc a duplicate of the other document?

Reply:
Hmm yes I did steal this, I'll update line 2

Reply:
I meant all the script looks to be the same as the other file

Reply:
Some is taken from [this script](https://github.com/we-discover/public/blob/master/entity_rsa_checker/main.gs), but that's just to get the ad group IDs where there are no RSAs. The rest of the functionality is new

Reply:
Whoops, yes thank you for spotting that!! I'd accidentally overwritten it.